CHANGELOG.rst

```diff
@@ -1,5 +1,49 @@
 Changelog
 =========
+1.8.0 (2020-04-16)
+------------------
+MLflow 1.8.0 includes several major features and improvements:
+
+Features:
+
+- Csteegz/update azml (#2711, @akshaya-a)
+- Case sensitivity and more tests for LIKE and ILIKE (#2708, @mparkhe)
+- Change line smoothing on Exponential Moving Average (#2620, @Valentyn1997)
+- Add support for spacy flavor (#2242, @arocketman)
+- [UI] Separate selection checkbox into its own column (#2559, @harupy)
+- highlight changes in CompareRunView (#2565, @gabrielbretschner)
+- Add run comparison in model registry (#2537, @juntai-zheng)
+- RFC: Allow string experiment IDs in the UI (#2568, @jonas)
+- Passing args to docker run CLI (#2608, @ksanjeevan)
+- - added windows support for pushing images to aws ecr (#2500, @AndreyBulezyuk)
+
+Bug fixes and documentation updates:
+
+- Fix CLI summary message in mlflow azureml build_image CLI (#2712, @dbczumar)
+- update flower example (#2647, @tfurmston)
+- Let curl follow redirects (fix #2691) (#2699, @HiromuHota)
```

Comment:
For these bug fixes / doc updates, it'd be good to clearly state what use case is benefitted by the bugfix. For example, in this case, from the description in #2691 it looks like we fixed a bug in `mlflow models build-docker`, so we could say something like `"Fixed pip not found error while packaging models via mlflow models build-docker (#2699, @HiromuHota)"`

See https://github.com/mlflow/mlflow/blob/master/CHANGELOG.rst#170-2020-03-02 for more examples on wording from previous changelogs

Reply:
Since there are a number of these in this release (& users likely are more directly-impacted by bugfixes than documentation updates), it's probably worth it to split them out into separate `Bug fixes` & `Documentation updates` sections, with bug fixes listed first.